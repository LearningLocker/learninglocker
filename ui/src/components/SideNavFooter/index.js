import React from 'react';
import styled from 'styled-components';

const Footer = styled.div`
  color: #999;

  @media screen and (max-height: 765px) {
    width: 150px;
  }
`;

export default () => (
  <Footer>
    <a href="https://ht2ltd.zendesk.com/hc/en-us/categories/115000129989-Learning-Locker">Help centre</a>
    <div>Powered by</div>
    <a href="http://learninglocker.net" target="_blank" rel="noopener noreferrer">Learning Locker Cloud</a>
  </Footer>
);
