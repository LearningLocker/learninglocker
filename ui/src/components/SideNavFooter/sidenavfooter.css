.sidenav-footer {
  color: #999;
}

@media screen and (max-height: 765px) {
  .sidenav-footer {
    width: 150px;
  }
}
