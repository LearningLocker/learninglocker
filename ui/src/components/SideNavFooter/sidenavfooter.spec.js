import React from 'react';
import 'jest-styled-components';
import renderer from 'react-test-renderer';
import SideNavFooter from './index.js';

describe('SideNavFooter', () => {
  test('should render', () => {
    const sideNav = renderer
      .create(<SideNavFooter />)
      .toJSON();

    expect(sideNav).toMatchSnapshot();
  });
});
