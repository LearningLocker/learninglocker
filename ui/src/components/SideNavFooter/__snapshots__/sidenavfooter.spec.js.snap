// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideNavFooter should render 1`] = `
.c0 {
  color: #999;
}

@media screen and (max-height:765px) {
  .c0 {
    width: 150px;
  }
}

<div
  className="c0"
>
  <a
    href="https://ht2ltd.zendesk.com/hc/en-us/categories/115000129989-Learning-Locker"
  >
    Help centre
  </a>
  <div>
    Powered by
  </div>
  <a
    href="http://learninglocker.net"
    rel="noopener noreferrer"
    target="_blank"
  >
    Learning Locker Cloud
  </a>
</div>
`;
